{{cookiecutter.project_slug}}/.github/workflows/precommit.yml

```diff
@@ -0,0 +1,50 @@
+name: Formatting & compliance
+
+on:
+  push:
+    branches:
+    - actions
+  pull_request:
+    branches:
+    - actions
```

Comment:
Change to `main`.

Reply:
yes, thanks. done. Also it could be that the default branch is named something else by the user. In that case either we add it to the templates of the cookicutter or the user has to change it by hand